static/openapi.yaml

```diff
@@ -1430,14 +1430,20 @@ components:
           description: A list of the categories that the project is in
         client_side:
           type: string
-          enum: [required, optional, unsupported]
+          enum: [required, optional, unsupported, unknown]
           description: The client side support of the project
           example: required
         server_side:
           type: string
-          enum: [required, optional, unsupported]
+          enum: [required, optional, unsupported, unknown]
```

Comment:
Are there still any projects with unknown sidedness? I can go through and get rid of any unknown sidedness and null slugs.

Reply:
Yes there are some, I don't have a complete list, but here is an example: https://api.modrinth.com/v2/project/gh42nn1h

Reply:
Do you think you can change it for the mods with null slugs and unknown side info? I would happily revert making slugs nullable and adding the unknown field to the enum.

Reply:
Should all be done now, let me know if you come across any more with null slugs or unknown sidedness

Reply:
Thank you! Will definitely do that.